src/sentry/api/bases/organization.py

```diff
@@ -389,23 +389,23 @@ def _filter_projects_by_permissions(
                 span.set_tag("mode", "force_global_perms")
                 return projects
 
+            # There is a special case for staff, where we want to fetch a single project (OrganizationStatsEndpointV2)
+            # or all projects (OrganizationMetricsDetailsEndpoint) in _admin. Staff cannot use has_project_access
+            # like superuser because it fails dueto staff having no scopes. The workaround is to create a lambda that
+            # mimics checking for active projects like has_project_access without further validation.
+            # NOTE: We must check staff before superuser or else _admin will fail when both cookies are active
+            if is_active_staff(request):
+                span.set_tag("mode", "staff_fetch_all")
+                proj_filter = lambda proj: proj.status == ObjectStatus.ACTIVE  # noqa: E731
             # Superuser should fetch all projects.
             # Also fetch all accessible projects if requesting $all
-            if is_active_superuser(request) or include_all_accessible:
+            elif is_active_superuser(request) or include_all_accessible:
```

Comment:
so staff technically has greater access than superuser? given the lambda functions

Reply:
kind of? I'm pretty sure `request.access.has_project_access` will also pass for all active projects for superuser. I think the only one that's different is the `include_all_accessible` check

Reply:
ahh i see